indico/web/flask/blueprints/legacy.py

```diff
@@ -55,8 +55,8 @@
     'adminLayout-setDefaultPDFOptions', 'adminLayout-styles', 'adminList',
     'adminList-switchNewsActive', 'adminMaintenance', 'adminMaintenance-pack', 'adminMaintenance-performPack',
     'adminMaintenance-performTmpCleanup', 'adminMaintenance-tmpCleanup', 'adminProtection', 'adminServices-ipbasedacl',
-    'adminServices-ipbasedacl_fagrant', 'adminServices-ipbasedacl_farevoke', 'adminServices-oauthAuthorized',
```

Comment:
you don't really have to worry about anything in this file - removing it is fine but keeping it also won't cause any problems. it's only a whitelist of things to generate legacy urls for, but if the original one is gone it'll never be looked up here anyway